internal/core/items/image.rs

```diff
@@ -37,6 +37,7 @@ pub struct ImageItem {
     pub height: Property<LogicalLength>,
     pub image_fit: Property<ImageFit>,
     pub image_rendering: Property<ImageRendering>,
+    pub colorize: Property<Brush>,
```

Comment:
I think the old colorize property should also be removed from `ClippedImage`.

Reply:
No, ClippedImage needs all the properties   (there is no inheritance or anything.)

Reply:
Ah indeed, apologies.